package.json

```
@@ -14,19 +14,19 @@
   ],
   "types": "lib/index.d.ts",
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.0.0"
   },
   "eslintConfig": {
     "extends": [
       "plugin:@hapi/module"
     ]
   },
-  "dependencies": {},
   "devDependencies": {
-    "@hapi/code": "8.x.x",
+    "@hapi/code": "^9.0.0",
     "@hapi/eslint-plugin": "*",
-    "@hapi/lab": "24.x.x",
-    "typescript": "~4.0.2"
+    "@hapi/lab": "25.0.0-beta.1",
```

Pending https://github.com/hapijs/lab/pull/1050
```suggestion
    "@hapi/lab": "^25.0.0",
```